drm/panel: panel-novatek-nt36532: Add 60Hz display mode

rmuxnet opened the pull request:
yes nga

rmuxnet committed:
drm/panel: panel-novatek-nt36532: Add 60Hz display mode

rmuxnet commented:
merge fast sur me want 60 hz

rmuxnet commented:
> will it burn my panel?

pizdiec no bro

luka177 commented:
@rmuxnet please stop unprofessional comminactions
And change the 1st commonent to clearly explain the change

rmuxnet committed:
Fix display mode parameters for NT36532 panel

rmuxnet committed:
Revise GPU and GMU OPP tables in sm8250.dtsi

Updated GPU and GMU operating points with new frequencies and levels.

rmuxnet committed:
Add 300MHz operating point to GMU OPP table

rmuxnet committed:
drm/panel: panel-novatek-nt36532: Fix 60Hz mode timings to match downstream DTS values

rmuxnet committed:
Optimize display modes for Novatek NT36532 panel

This update introduces optimized display modes for the Novatek NT36532 panel, enhancing performance across various refresh rates. The changes include adjustments to the vertical front porch to mitigate artifacts and maintain consistent pixel clock rates.

rmuxv2 committed:
pm8150b: feat: add charging limit

Added support for user-defined charge limiting in the PM8150B

rmuxv2 committed:
pm8150b: feat: add charging limit

pm8150b: feat: add charging limit

vipaoL commented:
Why there are pmic and gpu changes also? They have nothing to do with this PR. Why did you replace clear expressions and named constants with just numbers?

rmuxnet commented:
Why tf is it metging here wtf

Nuke it from here. I'm changing stuff as I'm gonna try adding some weird thing i saw
